library/core/src/future/future.rs

```diff
@@ -23,6 +24,7 @@ use crate::task::{Context, Poll};
 /// When using a future, you generally won't call `poll` directly, but instead
 /// `.await` the value.
 ///
+/// [`async`]: [../../core/keyword.async.html]
```

Comment:
```suggestion
[`async`]: ../../core/keyword.async.html
```

Reply:
Ooh, you are totally right. This of course shouldn't use the square brackets, thanks.